### DISCUSSION TITLE
Why adsninja is allowed for XDA?

### DISCUSSION THREAD

**Discussion by Rudxain** [easylist/easylist#20004] [Q&A]
https://github.com/easylist/easylist/blob/c7824cbc93bd4ddab6a41d2b21b6226383843cab/easylist/easylist_allowlist.txt#L80

I can't find any Issue/Discussion about this, and GH Git `blame` errors with a timeout.

I've completely blocked it:
https://github.com/Rudxain/uBO-rules/blob/c55c2041366a8e9fd2f11d9a42ce10b4f76dc9b0/rx.adb.txt#L5

And didn't notice any breakage

**Comment by Yuki2718** [comment]
https://github.com/easylist/easylist/commit/485aea939636fd15dc052104ccd55b9fe4296b77
blame works fine here, or you can use `git log -G "@@\|\|cdn\.adsninja\.ca"`

**Reply by Rudxain** [reply]
Thanks, but that still doesn't explain _why_

I've tried cloning the repo, but it took so long that it got a network error (my internet is ~2Mbps. Yes, **bits, not bytes** 😅)

**Reply by Yuki2718** [reply]
That pretty well explains by itself. I reproduced the issue.

**Reply by Rudxain** [reply]
I meant the decision. What's the rationale behind allow-listing that domain?

**Reply by Yuki2718** [reply]
We allow something if otherwise site's functionality is broken, period.

**Reply by Rudxain** [reply]
Fair enough 👍
